### PR TITLE
Feedback fixes related to texts, alignment and content

### PR DESCRIPTION
**Pull request by dbuzhorkaa**

### CODE REVIEW

File: data/en/community.json [hunk 1]
```diff
@@ -110,7 +110,14 @@
...
       "imageUrl": "/images/get-started/google-cloud-platform.png",
       "title": "Google Cloud Next ’19",
-      "description": "We will also be available at the OSS Office Hours on 04/10 at Google Cloud Next ‘19 SF at Moscone Center"
+      "description": "We will also be available at the OSS Office Hours on 04/10 at Google Cloud Next ‘19 SF at Moscone Center."
+    },
+    {
+      "imageUrl": "/images/get-started/data-hub.svg",
+      "title": "Cloud Data Fusion launched on GCP",
+      "description": "Cloud Data Fusion is a fully managed, cloud-native, enterprise data integration service, powered by CDAP, for quickly building and managing data pipelines on Google Cloud Platform. Visit the Google Cloud Console to try it out today.",
+      "linkTitle": "Google Cloud Console",
```
Comment (bdmogal) [inline data/en/community.json]: Visit Google Cloud Console

File: data/en/community.json [hunk 2]
```diff
@@ -110,7 +110,14 @@
     {
       "imageUrl": "/images/get-started/google-cloud-platform.png",
       "title": "Google Cloud Next ’19",
-      "description": "We will also be available at the OSS Office Hours on 04/10 at Google Cloud Next ‘19 SF at Moscone Center"
+      "description": "We will also be available at the OSS Office Hours on 04/10 at Google Cloud Next ‘19 SF at Moscone Center."
+    },
+    {
+      "imageUrl": "/images/get-started/data-hub.svg",
```
Comment (bdmogal) [inline data/en/community.json]: can we move this to the first

File: data/en/community.json [hunk 3]
```diff
@@ -110,7 +110,14 @@
     {
       "imageUrl": "/images/get-started/google-cloud-platform.png",
```
Comment (bdmogal) [inline data/en/community.json]: need to update this image

**Comment by bdmogal** [comment]
In the Evaluation section for cloud, change "Google Cloud Platform" to Google Cloud.

**Comment by bdmogal** [comment]
Same on the pipelines accelerators page.

File: data/en/community.json (hunk 1 above)
Comment (dbuzhorkaa) [reply data/en/community.json]: done

File: data/en/community.json (hunk 2 above)
Comment (dbuzhorkaa) [reply data/en/community.json]: done

File: data/en/community.json (hunk 3 above)
Comment (dbuzhorkaa) [reply data/en/community.json]: done

**Pull request merged**